modules/macros/src/test/scala-3/higherkindness/droste/derivation/DerivedTests.scala

```diff
@@ -0,0 +1,45 @@
+package higherkindness.droste.derivation
+
+import cats.{Foldable, Functor, Traverse}
+
+class FoldableTests {
```

Comment:
Do these tests come from kittens as well? do we want to add attribution?

Reply:
They do, and I thought I had - thanks